go.mod

```diff
@@ -4,17 +4,27 @@ go 1.16
 
 require (
 	cloud.google.com/go v0.82.0
+	github.com/DataDog/zstd v1.5.0 // indirect
+	github.com/cespare/xxhash/v2 v2.1.2 // indirect
+	github.com/cockroachdb/errors v1.8.6 // indirect
+	github.com/cockroachdb/logtags v0.0.0-20211118104740-dabe8e521a4f // indirect
+	github.com/cockroachdb/pebble v0.0.0-20211208200752-b10ca3a5c8f1 // indirect
+	github.com/cockroachdb/redact v1.1.3 // indirect
 	github.com/dn365/gin-zerolog v0.0.0-20171227063204-b43714b00db1
 	github.com/ghodss/yaml v1.0.0
 	github.com/gin-contrib/cors v1.3.1
 	github.com/gin-gonic/gin v1.7.4
 	github.com/golang-jwt/jwt/v4 v4.0.0
+	github.com/golang/snappy v0.0.4 // indirect
 	github.com/google/go-querystring v1.1.0
 	github.com/google/uuid v1.2.0
 	github.com/googleapis/gax-go v1.0.3
 	github.com/joho/godotenv v1.3.0
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/rotationalio/honu v0.2.3
+	github.com/klauspost/compress v1.13.6 // indirect
+	github.com/kr/pretty v0.3.0 // indirect
+	github.com/rogpeppe/go-internal v1.8.0 // indirect
+	github.com/rotationalio/honu v0.2.5-0.20211209153613-4aa259a30af1
```

Comment:
TODO: to get the tests to pass in this PR, I'm directly working off of the PR banch in Honu; however before we merge this PR we need to accurately set the Honu dependency to a tagged dependency. I'm thinking of resetting the `v0.2.4` dependency, but we can also bump it to `v0.2.5` if you think that's best.